_templates/layout.html

```diff
@@ -19,4 +19,11 @@
         a.appendChild(r);
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');	
 </script>
+<script type="text/javascript">
```

Comment:
I don't think you need to do this.
Simply put this in the head section: `<base target="_blank">` 
If there is a specific URL that shouldn't open on a different tab, add `_self` to it.  :-)

Reply:
Whoa, that was genius!!!